db/migrate/20150922141113_devise_create_admins.rb

```diff
@@ -0,0 +1,41 @@
+class DeviseCreateAdmins < ActiveRecord::Migration
+  def change
+    create_table(:admins) do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+
+      ## Recoverable
+      t.string   :reset_password_token
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.